components/MainScene.brs

```diff
@@ -25,9 +25,6 @@ sub init()
 
     ' create/set global fields with Channel dimensions (m.global.channelWidth/channelHeight)
     setChannelWidthHeightFromRootScene()
```

Comment:
Removing the `LoadingFlow` who then takes care of properly loading the component library? I think showing a pattern of how to load the component library (and ensure it's properly completed loading before doing anything with true[X]) is a useful pattern, but admittedly non essential. If you feel we can do away with it and simplify the example and still make it work, I'm ok with this.

Reply:
Oddly enough, this pattern is taken care of in the main scene.  Loading flow had nothing to do with the component library.  It would be nice to include the spinner, but thats not the focus of the app.